### PR TITLE
README for the base container improvements

### PR DESCRIPTION
**Pull request by dirkmueller**
As requested by the UX/UI work - include badges and better description for the LTSS / FIPS container.

### CODE REVIEW

File: src/bci_build/package/base-fips/README.md.j2
Comment (dcermak) [inline src/bci_build/package/base-fips/README.md.j2:1]:
```suggestion
```

File: src/bci_build/package/__init__.py
Comment (dcermak) [inline src/bci_build/package/__init__.py:1041]: Either capitalize both container and image or neither
```suggestion
            # The {self.title} Container Image
```

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dcermak) [inline src/bci_build/package/templates/access_protected_images.j2:3]:
```suggestion
This Container Image requires a subscription from SUSE to access. In order
```

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dcermak) [inline src/bci_build/package/templates/access_protected_images.j2:6]:
```suggestion
### Use the system registration of your host system
```

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dcermak) [inline src/bci_build/package/templates/access_protected_images.j2:8]: Sounds like German sentence order to me, afaik the sentences should be the other way around.

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dcermak) [inline src/bci_build/package/templates/access_protected_images.j2:15]:
```suggestion
set -o history
```

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dcermak) [inline src/bci_build/package/templates/access_protected_images.j2:14]:
```suggestion
 echo PASSWORD | podman login -u USERNAME --password-stdin registry.suse.com
```

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dcermak) [inline src/bci_build/package/templates/access_protected_images.j2:18]:
```suggestion
### Use a separate SUSE Customer Center registration code
```

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dcermak) [inline src/bci_build/package/templates/access_protected_images.j2:19]:
```suggestion
You can use a valid SUSE Customer Center registration code to log in to the registry, if the host system is not registered with the SUSE Customer Center, :
```

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dcermak) [inline src/bci_build/package/templates/access_protected_images.j2:26]:
```suggestion
The user parameter in this case is the verbatim string `regcode` and `SCC_REGISTRATION_CODE` is the actual registration code obtained from SUSE.
```

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dcermak) [inline src/bci_build/package/templates/access_protected_images.j2:28]:
```suggestion
### Use the organization mirroring credentials
```

File: src/bci_build/package/tomcat/README.md.j2
Comment (dcermak) [inline src/bci_build/package/tomcat/README.md.j2:72]: please add a big **fat** warning to the tomcat container package list, that the tomcat package **must** come first or else this will break

File: src/bci_build/package/tomcat/README.md.j2
Comment (dcermak) [inline src/bci_build/package/tomcat/README.md.j2:73]: Actually, this whole section needs to go anyway, as the samples are not shipped in the repo
```suggestion
```

File: src/bci_build/package/minimal/README.md.j2
Comment (dcermak) [inline src/bci_build/package/minimal/README.md.j2:9]: Hm, I would rather formulate it like this (or something along those lines) to prevent this sounding like "we're shipping an image that will most likely break for you".
```suggestion
However, this approach is not recommended and should only be employed by experienced users.
```

File: src/bci_build/package/__init__.py
Comment (dirkmueller) [reply src/bci_build/package/__init__.py:1041]: @dcermak you do realize that I just moved the line around and it has been that way all the time?

File: src/bci_build/package/minimal/README.md.j2
Comment (dirkmueller) [reply src/bci_build/package/minimal/README.md.j2:9]: this is the official documentation approved text, I just rewrapped it. see https://documentation.suse.com/container/all/single-html/Container-guide/index.html#id-sle-bci-minimal-when-you-do-not-need-zypper

let's do a documentation cleanup in a followup PR, it's difficult enough to get one semantically coherent change together.

File: src/bci_build/package/__init__.py
Comment (dcermak) [reply src/bci_build/package/__init__.py:1041]: Ok, so let's not fix it then

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dirkmueller) [reply src/bci_build/package/templates/access_protected_images.j2:3]: I've updated it to the wording in the container guide.

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dirkmueller) [reply src/bci_build/package/templates/access_protected_images.j2:8]: this is the official text from the current documentation. I'm not changing it in this PR. lets do this in a followup where documentation team can weigh in on the changes.

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dirkmueller) [reply src/bci_build/package/templates/access_protected_images.j2:19]: this is the official text from the existing container documentation, I'm not changing it in this PR. lets do this ina  followup where the documentation team can weigh in.

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dirkmueller) [reply src/bci_build/package/templates/access_protected_images.j2:26]: added

File: src/bci_build/package/tomcat/README.md.j2
Comment (dirkmueller) [reply src/bci_build/package/tomcat/README.md.j2:72]: tomcat is already installed in the tomcat container, no warning necessary.

File: src/bci_build/package/tomcat/README.md.j2
Comment (dirkmueller) [reply src/bci_build/package/tomcat/README.md.j2:73]: the ones I listed are shipped in the repo: 

```
S | Name                   | Type    | Version               | Arch   | Repository
--+------------------------+---------+-----------------------+--------+-----------
  | tomcat-admin-webapps   | package | 9.0.87-150200.65.1    | noarch | SLE_BCI
  | tomcat-docs-webapp     | package | 9.0.87-150200.65.1    | noarch | SLE_BCI
  | tomcat-webapps         | package | 9.0.87-150200.65.1    | noarch | SLE_BCI
  | tomcat10-admin-webapps | package | 10.1.20-150200.5.22.2 | noarch | SLE_BCI
  | tomcat10-docs-webapp   | package | 10.1.20-150200.5.22.2 | noarch | SLE_BCI
  | tomcat10-webapps       | package | 10.1.20-150200.5.22.2 | noarch | SLE_BCI
```

Comment (dirkmueller) [reply src/bci_build/package/tomcat/README.md.j2:73]: also, please apply your tomcat related review to the #1073  PR where this commit originates from. this one is supposed to come after #1073 is reviewed and merged.

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dmpop) [inline src/bci_build/package/templates/access_protected_images.j2:3]:
```suggestion
To access this image container image requires a valid SUSE subscription. In order
```

File: src/bci_build/package/tomcat/README.md.j2
Comment (dmpop) [inline src/bci_build/package/tomcat/README.md.j2:35]:
```suggestion
host. This user does not have write access to the mounted directory. To avoid
```

File: src/bci_build/package/busybox/README.md.j2
Comment (dirkmueller) [reply src/bci_build/package/busybox/README.md.j2:4]: do you mean something like "most basic tools provided by Coreutils" ? or a really concrete list? the concrete list is subject to change, it would be maintenance effort to keep this up to date with busybox updates that potentially introduce new things.

File: src/bci_build/package/tomcat/README.md.j2
Comment (dcermak) [reply src/bci_build/package/tomcat/README.md.j2]: Well, you've merged #1073 without waiting for my review there :shrug:

I just recall that you said that the `*-webapps` packages are in the webscripting module and should not be in SLE_BCI. I explicitly added them to the SLE_BCI repo, hence why they are available.

Comment (dirkmueller) [reply src/bci_build/package/tomcat/README.md.j2]: @dcermak it was just moving code around, we don't need to have two reviews for such simple changes. An isolated PR for this matter would have been sufficient. 

Anyhow, I do remember that `*-docs-webapp` should have been removed. I'll send a PR for that.

Comment (dirkmueller) [reply src/bci_build/package/tomcat/README.md.j2]: moved to #1084

File: src/bci_build/package/__init__.py
Comment (dirkmueller) [reply src/bci_build/package/__init__.py:1381]: it was in the ApplicationStackContainer class? I moved it into an inline implementation on the baseclass now.

File: src/bci_build/package/busybox/README.md.j2
Comment (alexandrevicenzi) [reply src/bci_build/package/busybox/README.md.j2:4]: > The SLE BCI-BusyBox image comes with tiny versions of many common UNIX utilities provided by the BusyBox project.

This is more in line with what BusyBox is in reality, I took this bit from "BusyBox combines tiny versions of many common UNIX utilities into a single small executable."

File: src/bci_build/package/templates/access_protected_images.j2
Comment (alexandrevicenzi) [inline src/bci_build/package/templates/access_protected_images.j2:3]: > To access this image container image requires a valid SUSE subscription.

Very confusing sentence, missing a comma somewhere?

Comment (dirkmueller) [reply src/bci_build/package/templates/access_protected_images.j2:3]: reworded, good catch

File: src/bci_build/package/templates/access_protected_images.j2
Comment (dirkmueller) [reply src/bci_build/package/templates/access_protected_images.j2:4]: I don't want to do a numbered listing here, as it might be confusing others that they need to do all three as steps in order, while only one of them is sufficient. 

I've reworded it slightly

Comment (alexandrevicenzi) [reply src/bci_build/package/templates/access_protected_images.j2]: Looks good.

**Review by alexandrevicenzi — APPROVED** [review]

**Pull request merged**
